XamlControlsGallery/XamlControlsGallery.csproj

```diff
@@ -10,8 +10,8 @@
     <AssemblyName>AppUIBasics</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion>10.0.18843.0</TargetPlatformVersion>

```

Comment:
Since this feature requires an insider version of the SDK (or newer), I think we'll need to hold off pulling this into master until the SDK locks.

Reply:
The property is in WinUI, I'm curious what changes here required the retarget? Maybe it was a quirk thing? We can split the TargetPlatformVersion and the MaxTestedVersion if we need the app to be running against 19H1 quirks.

Reply:
I think we're (Steven and I) are going to create a separate 19H1 branch for the gallery app and I'll make the PR to that branch. afaik this wasn't a quirk, just a change in the framework to add this luminosity property.